uninstall/3.1/README.md

```diff
@@ -0,0 +1,37 @@
+# Uninstall Cloud Pak for Watson AIOps
+
+The scripts provided here can be used to uninstall Cloud Pak of Watson AIOps and delete resources created by it.
+
+## Prereqs
+- You need to have OC CLI installed
+- You have logged into your cluster using `oc login`
+
+## Getting started
+
+Clone this repo.
+```
+  git clone https://github.ibm.com/katamari/katamari-util.git 
+  cd katamari-util/uninstall/<version to uninstall>
+```
+
+## Preparing for uninstall
+
+There are resources created by Cloud Pak of Watson AIOps that you should review before deleting.  These include PVCs, secrets, CRDs, and config maps.  You might want to save some of these or consider not deleting them if you plan to reinstall.  You can check the `uninstall-cp4waiops-resource-groups.sh` file for a list of resources that the script will delete.  You can update this list with resources you don't want to delete.
```

Comment:
@kvijai82 should we mention the properties file as link [uninstall-cp4waiops-resource-groups.sh](uninstall-cp4waiops-resource-groups.sh) rather than just highlight it with back tick ?

Reply:
Updated.